.github/workflows/ci.yml

```diff
@@ -36,6 +38,35 @@ jobs:
       - run: cargo build --features serde
       - run: cargo test --features serde
 
+  test-features-default-no-grid-nor-flexbox:
+    name: "Test Suite [Features: std (no grid or flexbox)]"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@stable
+      - run: cargo build --no-default-features --features std
+      - run: cargo test --no-default-features --features std
+
+  # Flexbox
+  test-features-flexbox:
+    name: "Test Suite [Features: flexbox]"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@stable
+      - run: cargo build --no-default-features --features flexbox
+      - run: cargo test --no-default-features --features flexbox
+
+  # Grid
+  test-features-grid:
+    name: "Test Suite [Features: grid]"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@stable
+      - run: cargo build --no-default-features --features grid
+      - run: cargo test --no-default-features --features grid
```

Comment:
I'd suggest also enabling either the `std` or `alloc` feature for these tests. That's a more realistic usage in most cases.